lib/js/components/RichList/RichListItem/RichListItem.consts.ts

```diff
@@ -7,6 +7,7 @@ export type RichListItemType = typeof RICH_LIST_ITEM_TYPE[keyof typeof RICH_LIST
 export const RICH_LIST_ITEM_STATE = {
 	DEFAULT: 'default',
 	LOADING: 'loading',
+	DRAG: 'drag',
```

Comment:
BasicRichListItem changes content colors when is in hover. Look at textGroupState computed in BasicRLI